### PR TITLE
refactor(new-hope): Tabs tokens, variations, config

### PR DESCRIPTION
**Pull request by nikewht**
Вынесены токены в отдельный файл, исправлен конфиг, убрана вариация active

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.873.7083023187.swift'>color_default_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.873.7083023187.kt'>color_default_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.873.7083023187.ts'>color_default_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.873.7083023187.xml'>color_default_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.873.7083023187.swift'>color_plasma_b2c_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.873.7083023187.kt'>color_plasma_b2c_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.873.7083023187.ts'>color_plasma_b2c_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.873.7083023187.xml'>color_plasma_b2c_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.873.7083023187.swift'>color_plasma_web_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.873.7083023187.kt'>color_plasma_web_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.873.7083023187.ts'>color_plasma_web_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.873.7083023187.xml'>color_plasma_web_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.873.7083023187.swift'>color_sberHealth_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.873.7083023187.kt'>color_sberHealth_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.873.7083023187.ts'>color_sberHealth_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.873.7083023187.xml'>color_sberHealth_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.873.7083023187.swift'>color_sbermarket_business_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.873.7083023187.kt'>color_sbermarket_business_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.873.7083023187.ts'>color_sbermarket_business_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.873.7083023187.xml'>color_sbermarket_business_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.873.7083023187.swift'>color_sbermarket_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.873.7083023187.kt'>color_sbermarket_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.873.7083023187.swift'>color_sbermarket_metro_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.873.7083023187.kt'>color_sbermarket_metro_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.873.7083023187.ts'>color_sbermarket_metro_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.873.7083023187.xml'>color_sbermarket_metro_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.873.7083023187.ts'>color_sbermarket_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.873.7083023187.swift'>color_sbermarket_selgros_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.873.7083023187.kt'>color_sbermarket_selgros_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.873.7083023187.ts'>color_sbermarket_selgros_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.873.7083023187.xml'>color_sbermarket_selgros_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.873.7083023187.swift'>color_sbermarket_wlbusiness_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.873.7083023187.kt'>color_sbermarket_wlbusiness_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts'>color_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.873.7083023187.xml'>color_sbermarket_wlbusiness_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.873.7083023187.xml'>color_sbermarket_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.873.7083023187.swift'>color_sberonline_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.873.7083023187.kt'>color_sberonline_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.873.7083023187.ts'>color_sberonline_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.873.7083023187.xml'>color_sberonline_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.873.7083023187.swift'>color_sberprime_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.873.7083023187.kt'>color_sberprime_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.873.7083023187.ts'>color_sberprime_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.873.7083023187.xml'>color_sberprime_xml--canary.873.7083023187.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.873.7083023187.swift'>color_stylesSalute_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.873.7083023187.kt'>color_stylesSalute_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.873.7083023187.ts'>color_stylesSalute_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.873.7083023187.xml'>color_stylesSalute_xml--canary.873.7083023187.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.873.7083023187.ts'>shadow_default_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.873.7083023187.ts'>shadow_plasma_b2c_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.873.7083023187.ts'>shadow_plasma_web_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.873.7083023187.ts'>shadow_sberHealth_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.873.7083023187.ts'>shadow_sbermarket_business_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.873.7083023187.ts'>shadow_sbermarket_metro_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.873.7083023187.ts'>shadow_sbermarket_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.873.7083023187.ts'>shadow_sbermarket_selgros_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts'>shadow_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.873.7083023187.ts'>shadow_sberonline_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.873.7083023187.ts'>shadow_sberprime_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.873.7083023187.ts'>shadow_stylesSalute_react-native--canary.873.7083023187.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.873.7083023187.ts'>borderRadius_default_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.873.7083023187.ts'>borderRadius_plasma_b2c_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.873.7083023187.ts'>borderRadius_plasma_web_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.873.7083023187.ts'>borderRadius_sberHealth_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.873.7083023187.ts'>borderRadius_sbermarket_business_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.873.7083023187.ts'>borderRadius_sbermarket_metro_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.873.7083023187.ts'>borderRadius_sbermarket_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.873.7083023187.ts'>borderRadius_sbermarket_selgros_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.873.7083023187.ts'>borderRadius_sberonline_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.873.7083023187.ts'>borderRadius_sberprime_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.873.7083023187.ts'>borderRadius_stylesSalute_react-native--canary.873.7083023187.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.873.7083023187.ts'>spacing_default_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.873.7083023187.ts'>spacing_plasma_b2c_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.873.7083023187.ts'>spacing_plasma_web_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.873.7083023187.ts'>spacing_sberHealth_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.873.7083023187.ts'>spacing_sbermarket_business_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.873.7083023187.ts'>spacing_sbermarket_metro_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.873.7083023187.ts'>spacing_sbermarket_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.873.7083023187.ts'>spacing_sbermarket_selgros_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts'>spacing_sbermarket_wlbusiness_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.873.7083023187.ts'>spacing_sberonline_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.873.7083023187.ts'>spacing_sberprime_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.873.7083023187.ts'>spacing_stylesSalute_react-native--canary.873.7083023187.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.873.7083023187.swift'>typo_mage_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.873.7083023187.kt'>typo_mage_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.873.7083023187.ts'>typo_mage_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.873.7083023187.swift'>typo_plasma_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.873.7083023187.kt'>typo_plasma_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.873.7083023187.ts'>typo_plasma_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.873.7083023187.swift'>typo_ruler_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.873.7083023187.kt'>typo_ruler_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.873.7083023187.ts'>typo_ruler_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.873.7083023187.swift'>typo_sage_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.873.7083023187.kt'>typo_sage_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.873.7083023187.ts'>typo_sage_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.873.7083023187.swift'>typo_sbermarket_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.873.7083023187.kt'>typo_sbermarket_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.873.7083023187.ts'>typo_sbermarket_react-native--canary.873.7083023187.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.873.7083023187.swift'>typo_soulmate_ios-swift--canary.873.7083023187.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.873.7083023187.kt'>typo_soulmate_kotlin--canary.873.7083023187.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.873.7083023187.ts'>typo_soulmate_react-native--canary.873.7083023187.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-asdk@0.10.1-canary.873.7083023187.0
  npm install @salutejs/plasma-b2c@1.250.1-canary.873.7083023187.0
  npm install @salutejs/plasma-new-hope@0.17.1-canary.873.7083023187.0
  npm install @salutejs/plasma-web@1.250.1-canary.873.7083023187.0
  # or 
  yarn add @salutejs/plasma-asdk@0.10.1-canary.873.7083023187.0
  yarn add @salutejs/plasma-b2c@1.250.1-canary.873.7083023187.0
  yarn add @salutejs/plasma-new-hope@0.17.1-canary.873.7083023187.0
  yarn add @salutejs/plasma-web@1.250.1-canary.873.7083023187.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!

  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-873/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-873/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-873/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-873/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-873/new-hope-storybook/
asdk storybook: http://plasma.sberdevices.ru/pr/pr-873/asdk-storybook/

File: packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts [hunk 1]
```diff
@@ -12,22 +12,22 @@ export const config = {
...
+                --plasma-tabs-font-color-hover: var(--text-accent);
             `,
         },
         active: {
             true: css`
                 --plasma-tabs-underline-color: var(--text-primary);
-                --plasma-tabs-color: var(--text-primary);
-                --plasma-tabs-color-hover: var(--text-primary);
+                --plasma-tabs-font-color: var(--text-primary);
+                --plasma-tabs-font-color-hover: var(--text-primary);
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts:30]: а можешь, пожалуйста, токены, которые ты вынес в отдельный файл перенести и сюда тоже?

File: packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts [hunk 2]
```diff
@@ -12,22 +12,22 @@ export const config = {
                 --plasma-tabs-underline-color: var(--surface-solid-tertiary);
                 --plasma-tabs-underline-height: 2px;
                 --plasma-tabs-font-weight: 500;
-                --plasma-tabs-color: var(--text-accent);
+                --plasma-tabs-font-color: var(--text-accent);
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts:15]: *не критично*

кажется, что не обязательно добавлять `font`, т.к. именно для фона какого-то элемента (в данном случае таба) используется свойство background, и выглядело бы оно тогда --plasma-tabs-background

File: packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const tokens = {
+    cursor: '--plasma-tabs-cursor',
+    underlineHeight: '--plasma-tabs-underline-height',
+    underlineColor: '--plasma-tabs-underline-color',
+    fontWeight: '--plasma-tabs-font-weight',
+    fontColor: '--plasma-tabs-font-color',
+    fontColorHover: '--plasma-tabs-font-color-hover',
+};
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts:8]: 👍

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!

  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-873/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-873/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-873/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-873/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-873/new-hope-storybook/
asdk storybook: http://plasma.sberdevices.ru/pr/pr-873/asdk-storybook/

File: packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts [hunk 4]
```diff
@@ -1,33 +1,34 @@
 // generated by theme-builder { version } { plasma-hope-version??? }
-
 import { css } from '@linaria/core';
 
+import { tokens } from '../../../../components/Tabs/TabItem.tokens';
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts:4]: здесь лучше забрать токены из индекса, по аналогии с Dropdown:

```ts
import { dropdownTokens } from '../../../../../components/Dropdown';
```

File: packages/plasma-new-hope/src/components/Tabs/index.ts [hunk 5]
```diff
@@ -1 +1,2 @@
 export { tabsRoot, tabsConfig } from './Tabs';
+export { tabItemRoot, tabItemConfig } from './TabItem';
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/components/Tabs/index.ts:2]: тут ещё нужно добавить ре-экспорт токенов

```ts
export { classes as dropdownClasses, tokens as dropdownTokens } from './Dropdown.tokens';
```

File: packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts (hunk 3 above)
Comment (neretin-trike) [reply packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts:8]: а вообще подумал, что лучше эти токены перенести в общий файлик - `Tabs.tokens` и забирать из одного места во всех компонентах

File: packages/plasma-new-hope/src/components/Tabs/TabItem/variations/_disabled/base.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { css } from '@linaria/core';
+
+import { tokens } from '../../../TabItem.tokens';
+
+export const base = css`
+    :hover {
+        cursor: var(${tokens.cursor});
+    }
+`;
```
Comment (neretin-trike) [inline packages/plasma-new-hope/src/components/Tabs/TabItem/variations/_disabled/base.ts:9]: кажется, что это должно быть не в `disabled`?  🤔

File: packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts (hunk 1 above)
Comment (nikewht) [reply packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts]: Поправил

File: packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts (hunk 3 above)
Comment (nikewht) [reply packages/plasma-new-hope/src/components/Tabs/TabItem.tokens.ts]: Поправил

File: packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts (hunk 4 above)
Comment (nikewht) [reply packages/plasma-new-hope/src/examples/sds_engineer/components/Tabs/TabItem.config.ts]: Поправил

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!

  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-873/

**Review by neretin-trike — APPROVED** [review]

**Comment by Yeti-or** [comment]
🎱

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.10.1-dev.0, @salutejs/plasma-b2c@1.250.1-dev.0, @salutejs/plasma-new-hope@0.17.1-dev.0, @salutejs/plasma-temple-docs@0.187.1-dev.0, @salutejs/plasma-tokens-native@1.16.1-dev.0, @salutejs/plasma-ui-docs@0.236.1-dev.0, @salutejs/plasma-web-docs@0.204.1-dev.0, @salutejs/plasma-web@1.250.1-dev.0, @salutejs/plasma-website@0.212.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.14.0, @salutejs/plasma-b2c@1.254.0, @salutejs/plasma-hope@1.248.0, @salutejs/plasma-new-hope@0.21.0, @salutejs/plasma-temple-docs@0.191.0, @salutejs/plasma-tokens-native@1.17.0, @salutejs/plasma-ui-docs@0.240.0, @salutejs/plasma-web-docs@0.208.0, @salutejs/plasma-web@1.254.0, @salutejs/plasma-website@0.216.0 🚀

**Pull request merged**